.github/workflows/sdk.yml

```diff
@@ -0,0 +1,37 @@
+name: sdk
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
This is *only* here so we can test the workflow as part of this PR.

We need to remove it before we merge, otherwise we'll be accidentally releasing versions on PR instead of after approval.

Reply:
I realized that, actually, keeping this will be good so long as we make sure it's a `dry-run`

In fact, once merged I think we should make this a required action, which means that if a PR breaks the SDK build it would flag an issue.